internal/checker/interpreter.go

```diff
@@ -1,22 +1,34 @@
 package checker
 
 import (
+	"os"
+	"strings"
+
 	"github.com/avorty/spito/pkg/shared"
 	"github.com/yuin/gopher-lua"
 )
 
+const rulesetDirConstantName = "@ruleset"
+
 func ExecuteLuaMain(script string, importLoopData *shared.ImportLoopData) (bool, error) {
 	L := lua.NewState(lua.Options{SkipOpenLibs: true})
 	defer L.Close()
 
+	currentDirectory, err := os.Getwd()
```

Comment:
It should not be working directory but directory of ruleset (notice that ruleset should be stored in `~/.local/state/spito-rules` or other location)

Reply:
It's a good point, although as @BaderBC has stated in #56 the user should be allowed to check his/her rules in any directory, such as `/home/example/projects/my-ruleset`, because `~/.local/state/spito-rules` is a directory used only for fetching and storing rules.

Reply:
I believe it should take a different location when the rule is executed locally in the "development" path (e.g. `/home/user/my-ruleset`) and different if the rule is imported or executed using non development check command.